src/terminology.md

```diff
@@ -6,42 +6,45 @@ Definitions are sorted alphabetically.
 
 ## Terms
 
-<span id="definition-assured-finality">**Assured Finality**</span>: A protocol property that assures that transactions cannot be reverted by that protocol. As with all protocol guarantees, a protocol assumes certain conditions must be met. A transaction may either be final or not: transactions which are not final may not become final, whereas once transactions do achieve finality they retain that property indefinitely (so long as protocol requirements are met).
+<!-- Unlike true html, it seems `mdbook` requires the span tags to be empty and immediately pre-fix the intended anchor target. -->
```

Comment:
`mdbook` is actually doing the Right Thing, and I should have realised it. The "**" markup is used for bold text in Markdown, but has no special meaning in HTML. An HTML container (such as `span`) embedded in Markdown should have its contents parsed as the Markdown-supported subset of HTML, not as Markdown. Therefore the rendering with non-bold "**Assured Finality**", etc. is correct. You would need `<span id="definition-assured-finality"><b>Assured Finality</b></span>` (or `<strong>`, I'm not exactly sure which tags are supported). In any case this is fine for now. I will fix the comment in my next PR.